src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -338,6 +339,12 @@
 						"\tevent-projectile is arrow",
 						"\tdelete event-projectile")
 				.since("1.0");
+		
+		if(Skript.classExists("com.destroystokyo.paper.event.entity.ProjectileCollideEvent")) Skript.registerEvent("Projectile Collide", SimpleEvent.class, ProjectileCollideEvent.class, "projectile collide")
```

Comment:
More of a stylistic change, but you can put the check and registration on separate lines? Helps with readability and avoiding long lines 😃

Reply:
Done 👍

Reply:
Not done?

Reply:
The more I use git the less I understand it